spec/data/master.yaml

```diff
@@ -0,0 +1,167 @@
+---
```

Comment:
it might make sense to use less data in fixtures since this makes it more readable.